content/docs/azure/deploy/install-kubeflow.md

```diff
@@ -78,11 +79,11 @@ Run the following commands to set up and deploy Kubeflow. The code below include
 
 1. Create user credentials. You only need to run this command once.
 
-        az aks get-credentials --name <NAME> --resource-group <RG>
+        az aks get-credentials -n <NAME> -g <RESOURCE_GROUP_NAME>
 
-1. Run the following commands to download the latest binary from the Kubeflow releases page (link = https://github.com/kubeflow/kubeflow/releases/).
+1. Run the following commands to download the latest kfctl binary from the Kubeflow releases page (links from https://github.com/kubeflow/kubeflow/releases). While writing this document, the latest release was https://github.com/kubeflow/kubeflow/releases/tag/v0.6.2)
```

Comment:
Remove the last closing bracket on this line, as it has no matching opening bracket. Also, the link is broken - probably because of the closing bracket.